pkg/test/integration/remote.go

```diff
@@ -37,13 +38,14 @@ func downloadPath(url, path string) (string, error) {
 	if _, err = hasher.Write([]byte(url)); err != nil {
```

Comment:
https://golang.org/pkg/hash/#Hash

Not introduced in this PR, but `hasher.Write` will never return an error so you could probably just use `//nolint` here.